tests/linalg_test.py

```diff
@@ -104,6 +104,48 @@ def testSlogdet(self, n, dtype, rng):
                             check_dtypes=True, tol=1e-3)
     self._CompileAndCheck(np.linalg.slogdet, args_maker, check_dtypes=True)
 
+  @parameterized.named_parameters(jtu.cases_from_list(
+      {"testcase_name": "_n={}".format(
+           jtu.format_shape_dtype_string((n,n), dtype)),
+       "n": n, "dtype": dtype, "rng": rng}
+      for n in [0, 4, 5, 50]
+      for dtype in float_types() + complex_types()
+      for rng in [jtu.rand_default()]))
+  # TODO(phawkins): enable when there is an eigendecomposition implementation
+  # for GPU/TPU.
+  @jtu.skip_on_devices("gpu", "tpu")
+  def testEig(self, n, dtype, rng):
+    self.skipTest("Test disabled until Jaxlib 0.1.15 is released") # TODO(phawkins)
+    args_maker = lambda: [rng((n, n), dtype)]
```

Comment:
Could test a leading dimension here too, right? (Since NumPy supports arg shapes of `(..., M, M)`.)